descriptors/fastsurfer/run_fastsurfer.sh.json

```diff
@@ -0,0 +1,372 @@
+{
+  "name": "run_fastsurfer.sh",
+  "tool-version": "2.3.3+bd65b36",
+  "description": "run_fastsurfer.sh takes a T1 full head image and creates segmentation using FastSurferVINN and surfaces using recon-surf",
+  "command-line": "run_fastsurfer.sh [SID] [SUBJECTS_DIR] [T1_INPUT] [FS_LICENSE] [ASEGDKT_SEGFILE] [VOX_SIZE] [SEG_ONLY] [SEG_LOG] [CONFORMED_NAME] [NORM_NAME] [T2_INPUT] [REG_MODE] [THREADS] [DEVICE] [VIEWAGG_DEVICE] [BATCH_SIZE] [PYTHON_CMD] [SURF_ONLY] [NO_BIASFIELD] [TAL_REG] [NO_ASEGDKT] [NO_CEREB] [CEREB_SEGFILE] [NO_HYPOTHAL] [QC_SNAP] [THREE_T] [PARALLEL] [IGNORE_FS_VERSION] [FSTESS] [FSQSPHERE] [FSAPARC] [NO_FS_T1] [NO_SURFREG] [ALLOW_ROOT] [VERSION]",
+  "schema-version": "0.5",
+  "container-image": {
+    "image": "deepmi/fastsurfer:cuda-v2.3.3",
```

Comment:
From the discussions here and offline, it seems like this might be better to use `cpu-v2.3.3` instead and let the user switch the GPU container instead if they choose.